crd/nodeinfo/manifests/acn.azure.com_nodeinfo.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
missing [`doc.go`](https://github.com/Azure/azure-container-networking/blob/master/crd/nodenetworkconfig/manifests/doc.go) in this directory that is required for the embed to work

Reply:
added